roles/docker/tasks/subtasks/daemon.yml

```diff
@@ -17,6 +17,30 @@
   with_items:
     - /etc/docker
 
+- name: Docker | Check if in a lxc container
```

Comment:
I'm hesitant to add these as is as we do not want to provide support for these in general. I'm more likely to add a way to override the required bits through the inventory rather than have these here.